### DISCUSSION TITLE
Slack channel for DBT docs

### DISCUSSION THREAD

**Discussion by filipjelenkovicFO** [dbt-labs/dbt-core#4751] [Community]
I would suggest Slack channel for DBT documentation. If we want to write some notes, changes, requests, missing parts, problems found etc...
For example until now (2 months using DBT) I have found broken links, not complete instructions, some use cases not explained outdated info etc. Nothing to important but it would save some time if that got fixed. Off course if you guys have someone to fix it :)

**Comment by runleonarun** [comment]
Hi @filipjelenkovicFO! Thank you for bringing this up! We will eventually have a place in the dbt community to chat about our product docs! We are just building a team right now so I want to wait until the space can get the attention it deserves. 

If you ever see any issues, our docs are open sourced, so you can either [raise an issue](https://github.com/dbt-labs/docs.getdbt.com/issues) or a PR to fix the problems.
